### PR TITLE
moving rxjs to an external

### PR DESCRIPTION
**Pull request by TheMcMurder**
Relies on https://github.com/react-microfrontends/shared-dependencies/pull/2

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -9,7 +9,11 @@ module.exports = webpackConfigEnv => {
     webpackConfigEnv
   });
 
-  return webpackMerge.smart(defaultConfig, {
+  const rxjsExternals = {
```
Comment (internettrans) [inline webpack.config.js:12]: Out of curiosity, why not just put this into the object on line 16 through 38

Comment (TheMcMurder) [reply webpack.config.js:12]: TBH? No reason.

Comment (TheMcMurder) [reply webpack.config.js:12]: I was playing around with how it worked and I did it this way just for fun. I can change it if you'd prefer it inline.

**Comment by TheMcMurder** [comment]
🤦‍♂ I saw the green checkmark and thought that was you saying you approved it. I'll make any necessary changes in another PR.

**Review by internettrans — COMMENTED** [review]
lgtm

**Pull request merged**